dev-libs/ferrisloki/ferrisloki-3.0.13-r3.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit flag-o-matic
+
+DESCRIPTION="Loki C++ library from Modern C++ Design"
+HOMEPAGE="http://www.libferris.com/"
+SRC_URI="mirror://sourceforge/witme/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86 ~amd64-linux ~x86-linux"
+IUSE="static-libs"
+
+RDEPEND=">=dev-libs/libsigc++-2.6:2"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-3.0.13-r3-configure-libsigc.patch
+)
```

Comment:
I decided to collapse all the file mangling from src_prepare() in the -r2 ebuild to a single patch.

Reply:
Good idea, and I think it is safer that way, cause it was piling up to a bad ending at some point.